scvi/model/base/_vaemixin.py

```diff
@@ -50,8 +50,11 @@ def get_marginal_ll(
         adata: Optional[AnnData] = None,
         indices: Optional[Sequence[int]] = None,
         n_mc_samples: int = 1000,
+        n_samples_per_pass: int = 25,
         batch_size: Optional[int] = None,
-    ) -> float:
+        observation_specific: Optional[bool] = False,
+        _adata: Optional[AnnData] = None,
```

Comment:
what is the purpose of this?

Reply:
see above for `_adata`
The option `observation_specific` allows to return observation-specific marginal estimates, i.e., estimates of p(x_i), instead of mean marginal estimates